pallets/loans/src/types/mod.rs

```diff
@@ -85,25 +85,45 @@ pub enum MutationError {
 	DiscountedCashFlowExpected,
 	/// Emits when a modification expect the loan to have an iternal pricing.
 	InternalPricingExpected,
+	/// Maturity extensions exceed max extension allowed.
+	MaturityExtendedToMuch,
```

Comment:
```suggestion
	MaturityExtendedTooMuch,
```

Reply:
Added a test too to check this is error is correctly generated 👍🏻